src/Dispatcher/MASA.Contrib.Dispatcher.IntegrationEvents.Dapr/Options/DispatcherOptions.cs

```diff
@@ -13,13 +13,63 @@ public string PubSubName
             {
                 throw new ArgumentNullException(nameof(_pubSubName));
             }
+
             _pubSubName = value;
         }
     }
 
+    /// <summary>
+    /// maximum number of retries
+    /// Local queue maximum number of retries
+    /// </summary>
+    public int LocalRetryTimes { get; set; } = 3;
+
+    /// <summary>
+    /// maximum number of retries
+    /// Default is 10
+    /// </summary>
+    public int MaxRetryTimes { get; set; } = 10;
+
+    /// <summary>
+    /// The interval at which db polls for failure messages.
+    /// Default is 60 seconds.
+    /// </summary>
+    public int FailedRetryInterval { get; set; } = 60;
+
+    /// <summary>
+    /// The interval at which the local queue is polled for failed messages.
+    /// Local queue does not rebuild after service crash
+    /// Default is 3 seconds.
+    /// </summary>
+    public int LocalFailedRetryInterval { get; set; } = 3;
+
+    /// <summary>
+    /// The size of a single event to be retried
```

Comment:
What's the mean?

Reply:
The number of retry messages obtained for each retry

Reply:
Maximum number of retries?